### PR TITLE
Async storage queries

### PR DESCRIPTION
**Pull request by ivancho277**

### CODE REVIEW

File: poker-app/Home.js [hunk 1]
```diff
@@ -1,32 +1,33 @@
...
+const storage = require("./components/AsyncStorageController.js");
 class HomeScreen extends Component {
     state = {
         gamesArray : []
     }
 
     componentDidMount() {
         this.setState({
-            gamesArray: this.retrieveData()
+            gamesArray: storage.retrieveData
```
Comment (yordanr) [inline poker-app/Home.js]: did you mean to execute storage.retrieveData()?

File: poker-app/Home.js [hunk 2]
```diff
@@ -1,32 +1,33 @@
...
+    //     console.log(JSON.parse(games));
+    //     return JSON.parse(games);
+    // }
 
     render() {
         return (
             <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
                 <Text>Welcome to Poker</Text>
-                <StatsBox games={this.retrieveData} />
+                <StatsBox games={storage.retrieveData} />
```
Comment (yordanr) [inline poker-app/Home.js]: you call retrieve.data when you have your state?

File: poker-app/Home.js [hunk 3]
```diff
@@ -1,32 +1,33 @@
...
                 <Button title="Game" style={{margin: '10px'}} onPress={() => this.props.navigation.navigate('Game')} />
-                <TouchableOpacity onPress={this.retrieveData}>
+                <TouchableOpacity onPress={storage.retrieveData}>
```
Comment (yordanr) [inline poker-app/Home.js]: same here? should state be modified?

File: poker-app/components/AsyncStorageController.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import { View, Text, AsyncStorage } from 'react-native';
```
Comment (yordanr) [inline poker-app/components/AsyncStorageController.js]: check if async storage is still supported. 
try import AsyncStorage from '@react-native-community/async-storage'
there was some other stuff. check instructions.

you can do it later though. not that important if you reference asyncstorage in this file only.

File: poker-app/components/AsyncStorageController.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+            console.log(`success storing ${data.calls}`);
+            return data;
+        }
+        catch (error) {
+            console.log("error saving data");
+        }
+    },
+
+    retrieveData: async function() {
+        let keys = await AsyncStorage.getAllKeys();
```
Comment (yordanr) [inline poker-app/components/AsyncStorageController.js]: use constant key you know your data resides under.

File: poker-app/Home.js (hunk 1 above)
Comment (ivancho277) [reply poker-app/Home.js]: yes i did, thx

File: poker-app/Home.js (hunk 2 above)
Comment (ivancho277) [reply poker-app/Home.js]: I will send the data from state not call retrieve data again

**Pull request merged**